src/NodaTime.Test/TimeZones/BclDateTimeZoneTest.cs

```diff
@@ -136,6 +162,13 @@ public void ForSystemDefault()
         [Test]
         public void DateTimeMinValueStartRuleExtendsToBeginningOfTime()
         {
+            // .NET Core on Unix loses data from rules provided to CreateCustomTimeZone :(
+            // (It assumes the rules have been created from tzif files, which isn't the case here.)
+            if (TestHelper.IsRunningOnDotNetCoreUnix)
```

Comment:
Is there an assume-like construct in NUnit? That might be clearer than `if (expr) Assert.Ignore()`.

(Also same pattern elsewhere.)

Reply:
Not out of the box. It looks like we could do it with some extra code though:
https://amido.com/blog/conditional-ignore-nunit-and-the-ability-to-conditionally-ignore-a-test/

I may have a look if I get some time, but I won't try to do it in this PR.